libkbfs/folder_branch_ops.go

```diff
@@ -6195,7 +6195,7 @@ func (fbo *folderBranchOps) GetUpdateHistory(ctx context.Context,
 		}
 		updateSummary := UpdateSummary{
 			Revision:  rmd.Revision(),
-			Date:      time.Unix(0, rmd.data.Dir.Mtime),
+			Date:      rmd.localTimestamp,
```

Comment:
This is the server's timestamp, adjusted for the local clock according to ping latency measurements: https://github.com/keybase/kbfs/blob/master/libkbfs/root_metadata.go#L948-L957